app/assets/stylesheets/components/_dob-form-field.scss

```diff
@@ -21,10 +21,20 @@
   }
 }
 
+.form-dob__fields {
```

Comment:
`fields` is a bit ambiguous I think.

Is this essentially a container for the inputs?